### PR TITLE
Add tests to ensure that both abs(0.0) and abs(-0.0) lead to 0.0

### PR DESCRIPTION
**Pull request by xuhdev**
As pointed out by @colesbury in https://github.com/pytorch/pytorch/pull/23579#discussion_r309798987

### CODE REVIEW

File: test/test_cuda.py [hunk 1]
```diff
@@ -1085,6 +1085,13 @@ def test_type_conversions_same_gpu(self):
         self.assertEqual(x.type(torch.int).get_device(), 1)
         self.assertEqual(x.to(torch.int).get_device(), 1)
 
+    def test_abs_zero(self):
```
Comment (vishwakftw) [inline test/test_cuda.py]: Do we even test for `abs`’s behavior on CUDA? I found no `test_abs` in this file.

Comment (xuhdev) [reply test/test_cuda.py]: It's done along with other operators at

https://github.com/pytorch/pytorch/blob/3314d60a759fa16d142049bed9be454e69c535db/test/test_cuda.py#L569

Comment (vishwakftw) [reply test/test_cuda.py]: Ok, thanks for letting me know.

**Review by mrshenli — APPROVED** [review]
LGTM!

**Pull request closed without merging**